figs/data/oss_impact/oss_impact.ipynb

```diff
@@ -81,7 +81,66 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 9,
+   "id": "f27c33df",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Country</th>\n",
+       "      <th>Alpha-2 code</th>\n",
+       "      <th>Alpha-3 code</th>\n",
+       "      <th>Numeric</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>217</th>\n",
+       "      <td>Taiwan (Province of China)</td>\n",
+       "      <td>TW</td>\n",
+       "      <td>TWN</td>\n",
+       "      <td>158</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                        Country Alpha-2 code Alpha-3 code  Numeric\n",
+       "217  Taiwan (Province of China)           TW          TWN      158"
```

Comment:
Same here

Reply:
Yeah, I noticed that... I am simply using public data sources (in this case [IBAN](https://www.iban.com/country-codes)) to translate between 2 letter codes (which GitHub used) and the 3 letter codes that the World Bank population data uses. 

In any case, the figure doesn't contain any labels -- only outlines on a map